config/main.py

```diff
@@ -1680,12 +1590,7 @@ def breakout(ctx, interface_name, mode, verbose, force_remove_dependencies, load
     del_intf_dict = {intf: del_ports[intf]["speed"] for intf in del_ports}
```

Comment:
Seems duplicate like. Can we please delete it?

Reply:
@samaity : Double confirmation, I agree we can delete this line, Should I go ahead.

Reply:
Yes, kindly do it. I have already deleted the duplicate line in the upstream code.